src/main/java/frc/robot/commands/RunIntakeForwards.java

```diff
@@ -1,41 +1,45 @@
 /*----------------------------------------------------------------------------*/
-/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
 /* Open Source Software - may be modified and shared by FRC teams. The code   */
 /* must be accompanied by the FIRST BSD license file in the root directory of */
 /* the project.                                                               */
 /*----------------------------------------------------------------------------*/
 
 package frc.robot.commands;
 
-import frc.robot.subsystems.ExampleSubsystem;
 import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.subsystems.Intake;
+import frc.robot.util.TunableNumber;
 
-/**
- * An example command that uses an example subsystem.
- */
-public class ExampleCommand extends CommandBase {
-  @SuppressWarnings({ "PMD.UnusedPrivateField", "PMD.SingularField", "unused" })
-  private final ExampleSubsystem m_subsystem;
+public class RunIntakeForwards extends CommandBase {
+
+  private TunableNumber setpoint = new TunableNumber("Intake Forwards/setpoint");
+  private final Intake intake;
 
   /**
-   * Creates a new ExampleCommand.
-   *
-   * @param subsystem The subsystem used by this command.
+   * Creates a new RunIntakeForwards.
+   * 
+   * @param Intake
    */
-  public ExampleCommand(ExampleSubsystem subsystem) {
-    m_subsystem = subsystem;
-    // Use addRequirements() here to declare subsystem dependencies.
-    addRequirements(subsystem);
+  public RunIntakeForwards(Intake intakeSub) {
+    intake = intakeSub;
+    addRequirements(intakeSub);
   }
 
   // Called when the command is initially scheduled.
   @Override
   public void initialize() {
+    setpoint.setDefault(0);
```

Comment:
This should also have a real default.
```suggestion
    setpoint.setDefault(-1);
```